src/app/[locale]/(protected)/history/history-client.tsx

```diff
@@ -0,0 +1,128 @@
+                </Flex>
+              }
+            ></Empty>
+
+            <Link href="/rest-client" scroll={false}>
+              <Button type="primary" style={{ marginTop: '20px' }}>
+                {t('goRestClient')}
+              </Button>
+            </Link>
+          </Flex>
+        ) : (
+          <Card
+            style={{
+              width: '100%',
+            }}
+          >
+            <Collapse
+              accordion
+              items={items.map((item) => ({
+                key: item.id,
+                label: (
+                  <Row gutter={16} wrap style={{ width: '100%' }}>
+                    <Col xs={12} md={4}>
+                      <Text strong>[{item.method}]</Text>
+                    </Col>
+
+                    <Col xs={12} md={4} style={{ textAlign: 'right' }}>
+                      <Text
+                        type={
+                          item.statusCode === 0
+                            ? 'danger'
+                            : item.statusCode >= 400
+                              ? 'danger'
+                              : item.statusCode >= 200 && item.statusCode < 300
+                                ? 'success'
+                                : 'secondary'
+                        }
+                      >
+                        {item.statusCode === 0 ? t('error') : item.statusCode}
+                      </Text>
+                    </Col>
+
+                    <Col
+                      xs={24}
+                      md={16}
+                      style={{ wordBreak: 'break-word', overflowWrap: 'break-word' }}
+                    >
+                      <Link
+                        href={item.appRouterURL}
+                        className={`link link-font_size_small ${themeValue === 'dark' ? 'link--dark' : 'link--light'}`}
+                        onClick={(e) => e.stopPropagation()}
+                        scroll={false}
+                      >
+                        {item.url}
+                      </Link>
+                    </Col>
+                  </Row>
+                ),
+                children: (
+                  <div>
+                    <p>
+                      <b>{t('timestamp')}:</b> {new Date(item.timestamp).toLocaleString()}
+                    </p>
+                    <p>
+                      <b>{t('duration')}:</b> {formatMs(item.durationMs)}
+                    </p>
+                    <p>
+                      <b>{t('requestSize')}:</b> {formatBytes(item.requestSize)}
+                    </p>
+                    <p>
+                      <b>{t('responseSize')}:</b> {formatBytes(item.responseSize)}
+                    </p>
+                    <p>
+                      <b>{t('error')}:</b>{' '}
+                      {item.errorDetails && item.errorDetails.trim() !== ''
+                        ? item.errorDetails
+                        : t('noError')}
```

Comment:
```suggestion
                      {item.errorDetails?.trim() && t('noError')}
```
- optional chaining 
- operator `AND` will fallback to `noError` if `errorDetails` is empty, since that will be as falsy value

Reply:
Thank you for the suggestion, but I think you meant `{item.errorDetails?.trim() || t('noError')}`. 

That code will either use `item.errorDetails?.trim()` or fallback to `t('noError')}` if `errorDtails` is empty.

I have applied the change.